app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+from app.players.player import Player
+from typing import List
```

Comment:
typing is part of the standard library, so it comes first. Then, the local application imports